src/SqlAsyncCollector.cs

```diff
@@ -237,22 +252,41 @@ private async Task UpsertRowsAsync(IEnumerable<T> rows, SqlAttribute attribute,
         /// to upsert to and returns the extra property names in a List.
         /// </summary>
         /// <param name="columns"> The columns of the table to upsert to </param>
+        /// <param name="row"> First row to get the column names when T is JObject </param>
```

Comment:
`Sample item used to get the column names when item is a JObject` seems clearer to me